addons/rating/views/rating_templates.xml

```diff
@@ -20,6 +20,7 @@
         <template id="rating_external_page_submit" name="Rate our Services">
             <t t-call="web.frontend_layout">
                 <div class="container mb-5 mt-4 o_rating_page_submit">
+                    <div class="oe_structure" id="oe_structure_rate_submit_header"/>
```

Comment:
The `id` is really necessary? 🤔

Reply:
yes, It is required because the studio makes this div(element) editable if id with prefix `oe_structure_`  without it it'll not make it editable! Thanks:slightly_frowning_face:

Reply:
ok thanks 🙂